### PR TITLE
weekly image cve patching from Sec Pipelines

### PR DESCRIPTION
**Pull request by HDYA**
weekly image cve patching from Sec Pipelines

### CODE REVIEW

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -170,6 +170,7 @@
       "downloadURL": "mcr.microsoft.com/oss/cilium/cilium:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
+        "1.12.5.1",
```
Comment (alexeldeib) [inline vhdbuilder/packer/components.json:173]: this is old, shouldn't be added back

File: vhdbuilder/packer/components.json [hunk 2]
```diff
@@ -200,7 +201,10 @@
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/autoscaler/cluster-proportional-autoscaler:*",
-      "amd64OnlyVersions": [],
+      "amd64OnlyVersions": [
+        "1.7.1-hotfix.20200403",
```
Comment (alexeldeib) [inline vhdbuilder/packer/components.json:205]: same

File: vhdbuilder/packer/components.json [hunk 3]
```diff
@@ -294,7 +298,9 @@
     {
       "downloadURL": "mcr.microsoft.com/oss/virtual-kubelet/virtual-kubelet:*",
       "amd64OnlyVersions": [],
-      "multiArchVersions": []
+      "multiArchVersions": [
+        "1.4.8"
```
Comment (alexeldeib) [inline vhdbuilder/packer/components.json:302]: same

**Comment by ganeshkumar5699** [comment]
From which file are these older, deleted versions being retrieved from?

File: vhdbuilder/packer/kube-proxy-images.json [hunk 4]
```diff
@@ -6,12 +6,16 @@
         "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:v*",
         "amd64OnlyVersions": [],
         "multiArchVersions": [
-          "1.24.9-hotfix.20230208.2",
           "1.24.10.3",
+          "1.24.9-hotfix.20230208.2",
           "1.25.5-hotfix.20230208",
```
[the remark this answers is not in the document]
Comment (HDYA) [reply vhdbuilder/packer/kube-proxy-images.json:11]: I don't think so? will update more on PR 3028 as this will be a duplicated PR

**Pull request closed without merging**